app/models/patient.rb

```diff
@@ -45,7 +45,8 @@ class Patient < ApplicationRecord
      primary_telephone_type
      secondary_telephone_type
      additional_planned_travel_type
-     case_status].each do |enum_field|
+     case_status
+     follow_up_reason].each do |enum_field|
```

Comment:
Only thing I'm wondering is if we also should be validating that `follow_up_note` is only set if `follow_up_reason` is set. I think to perfectly match the UI, we would want that to be the behavior. But it isn't as if setting the `follow_up_note` without `follow_up_reason` would actively cause any issues, so I'll leave it up to you to tell me it isn't worth it to validate that. Let me know what you think.

Reply:
You bring up a good point! After taking a look at the way it works in the UI again, it does seem like the user is required to select one of the follow up reasons (even if the follow up note is blank), so I will make the change you suggested along with not allowing blank as a value for follow up reason (nil is allowed though)

Reply:
https://github.com/SaraAlert/SaraAlert/pull/983/commits/55626eeef63be177ddce408de52fc9c96a84dc72#diff-94dcc014f62f0d9b6084ad98f82f74cd40c86b029ef1691914bdbc960c2f42dcR94